doc/api/http.md

```diff
@@ -2898,7 +2898,9 @@ class. The `ClientRequest` instance is a writable stream. If one needs to
 upload a file with a POST request, then write to the `ClientRequest` object.
 
 ```js
-const postData = querystring.stringify({
+const http = require("http")
+
+const postData = JSON.stringify({
```

Comment:
"application/x-www-form-urlencoded" payloads (which this example is sending) are not JSON.

Reply:
what do you suggest we do here?

Reply:
@mscdex  Should be application/json, right?

Reply:
Either that or revert back to using `querystring.stringify()`.